src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -51,45 +60,6 @@ internal partial class TdsParserStateObject
 
         internal bool _receivedColMetaData;      // Used to keep track of when to fire StatementCompleted  event.
```

Comment:
Could you make private this field and add the same property with get/set on it  as NetCore:
```csharp
        internal bool HasReceivedColumnMetadata
        {
            get => _receivedColMetaData;
            set => _receivedColMetaData= value;
        }
```